src/components/Filters/spec.js

```diff
@@ -133,6 +193,42 @@ describe('Filters component', () => {
       expect(component.find(FiltersSummary)).toHaveLength(0)
       component.unmount()
     })
+
+    it('should filter using "is" operator for both key and value properties of an object', () => {
+      const component = mount(
+        <Provider>
+          <Filters schema={schema} filters={[tagIsFilter]} />
+        </Provider>
+      )
+      expect(
+        component
+          .find('button')
+          .at(4)
+          .text()
+      ).toBe('Tag is rarity : 10')
+
+      const filteredByTagIs = SchemaSieve.filter(tagIsFilter, pokedex)
```

Comment:
There is a separate spec file for the filtering mechanism here https://github.com/balena-io-modules/rendition/blob/master/src/components/Filters/SchemaSieve.spec.js can you move this test into that file?